src/query/storages/common/cache/src/providers/disk_cache.rs

```diff
@@ -132,11 +133,165 @@ where C: Cache<String, u64, DefaultHashBuilder, FileSize>
         Ok(self)
     }
 
+    fn fuzzy_restart(self) -> result::Result<Self> {
+        fs::create_dir_all(&self.root)?;
+
+        let parallel_degree = std::thread::available_parallelism()
```

Comment:
This is the host physical CPU numbers not the docker/pod vCPU?

Reply:
Either case will be OK, since the task is not CPU intensive.